WikiPreview.Fluent.Plugin/WikiPreviewSearchOperation.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Blast.Core.Results;
+
+namespace WikiPreview.Fluent.Plugin
+{
+    class WikiPreviewSearchOperation
+    {
+
+        public class WikiSearchOperation : SearchOperationBase
+        {
+            protected internal WikiSearchOperation() : base("Open in Wikipedia",
+                "Opens the article on Wikipedia.",
+                "\uE71B")
+            {
+
+            }
+        }
+
+        public class WikiwandSearchOperation : SearchOperationBase
+        {
+            protected internal WikiwandSearchOperation() : base("Open in Wikiwand", "Opens the Wikipedia article in Wikiwand with a custom viewing experience.",
+                "\uE774")
+            {
+                
+            }
+        }
+
+        public class CopyUrlSearchOperation : SearchOperationBase
```

Comment:
Why aren't you using the built-in copy operation?

Reply:
Based on the output value of `switch`, if it is `null`, I'm executing the copy operation. Let me know on this.

Reply:
In the commit named `fixes`: I forgot to add the copy operation. It has been fixed in the following commit.